File: app/src/main/java/com/norvera/confession/data/models/InspirationEntry.java
package com.norvera.confession.data.models;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "INSPIRATION" )
public class InspirationEntry {
    @PrimaryKey
    @ColumnInfo(name = "_id")
    private final long id;

    @ColumnInfo(name="QUOTE")
    public final String text;

    @ColumnInfo(name="AUTHOR")
    public final String author;
    public InspirationEntry(long id, String text, String author) {
        this.id = id;
        this.text = text;
        this.author = author;
    }

    public long getId() {
        return id;
    }

    public String getText() {
        return text;
    }

    public String getAuthor() {
        return author;
    }
}

File: app/src/main/java/com/norvera/confession/data/models/InspirationEntry.kt
package com.norvera.confession.data.models

import android.os.Parcelable
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.android.parcel.Parcelize

@Entity(tableName = "INSPIRATION")
@Parcelize
data class InspirationEntry(
    @field:PrimaryKey
    @field:ColumnInfo(name = "_id")
    val id: Long, @field:ColumnInfo(name = "QUOTE")
    val text: String, @field:ColumnInfo(name = "AUTHOR")
    val author: String
) : Parcelable
